safe-room.lic

```diff
@@ -14,10 +14,15 @@ class SafeRoom
 
   def initialize
     arg_definitions = [
-      [{ name: 'force', regex: /force/i, description: 'Run healing logic even if there are no visible wounds.' }],
-      [{ name: 'skip', regex: /(s|skip)/i, description: 'Skip running to the safe room, useful for Empaths and Necros.' }],
+    arg_definitions = [
+      [ { name: 'force', regex: /force/i, description: 'Run healing logic even if there are no visible wounds.' } ],
```

Comment:
Would it be easier to just set `force` to `optional: true` instead of adding two more arg sets?

Reply:
I thought if we did this the force would require the skip first?